Updates for clean-up (#942)
MicrosoftDocs/biztalk-docs: his/core/working-with-dependent-appc-lus1.md

ms.topic: "article"
---
# Working with Dependent APPC LUs

> A dependent local APPC LU requires the support of a mainframe in order to communicate with a remote TP. Dependent APPC LUs cannot be used to communicate with IBM System is. Unlike independent APPC LUs, dependent APPC LUs only allow a single session per LU.  

Reviewer: ```suggestion
To communicate with a remote TP, a dependent local APPC LU requires the support of a mainframe. You can't use dependent APPC LUs to communicate with IBM System i computers. Unlike independent APPC LUs, dependent APPC LUs only allow a single session per LU.  
```
